<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tripPlanner</class>
 <widget class="QDialog" name="tripPlanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>591</height>
    </rect>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1041</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Plan">
     <attribute name="title">
      <string>Plan</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>301</width>
        <height>531</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>299</width>
         <height>529</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="planButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Plan</string>
      </property>
     </widget>
     <widget class="QComboBox" name="colName">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="warningLabel">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>100</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>170</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>3-Character Trip ID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="trip">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="window">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>20</y>
        <width>261</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="graphInfo">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Graph Info</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startTrip">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>212</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Trip !</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>MLB information</string>
     </attribute>
     <widget class="QTableView" name="souvenirView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1041</width>
        <height>451</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>470</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showALL">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>470</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Show ALL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showStadiums">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>500</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Team Name (A-Z)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showStadiums_2">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>470</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Stadiums (A-Z)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>470</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>American League</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>500</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>National  League</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>470</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Typology</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>470</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sort by:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>500</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Roof Type</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>530</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Date Opened</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>530</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Seat Capacity</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>470</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Dist to CF(great)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>500</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Dist to CF(least)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="colleges">
     <attribute name="title">
      <string>Distances</string>
     </attribute>
     <widget class="QTableView" name="collegeView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>541</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>tripPlanner</receiver>
   <slot>onTypologyClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>674</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>tripPlanner</receiver>
   <slot>onAmericanClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>775</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>778</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showALL</sender>
   <signal>clicked()</signal>
   <receiver>tripPlanner</receiver>
   <slot>onShowClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>tripPlanner</receiver>
   <slot>onNationalClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>765</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>765</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showStadiums_2</sender>
   <signal>clicked()</signal>
   <receiver>tripPlanner</receiver>
   <slot>onStadiumsClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>886</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>887</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showStadiums</sender>
   <signal>clicked()</signal>
   <receiver>tripPlanner</receiver>
   <slot>onTeamClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>906</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>902</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>tripPlanner</receiver>
   <slot>onSearchClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>tripPlanner</receiver>
   <slot>onRoofClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>688</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>tripPlanner</receiver>
   <slot>onDateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>674</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>tripPlanner</receiver>
   <slot>onSeatingClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>805</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>805</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>tripPlanner</receiver>
   <slot>onGreatClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1032</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>922</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>tripPlanner</receiver>
   <slot>onLeastClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>996</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>954</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>planButton</sender>
   <signal>clicked()</signal>
   <receiver>tripPlanner</receiver>
   <slot>onPlanClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startTrip</sender>
   <signal>clicked()</signal>
   <receiver>tripPlanner</receiver>
   <slot>onStartTrip()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>791</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>789</x>
     <y>627</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPlanClick()</slot>
  <slot>onDisplayClick()</slot>
  <slot>onStartClick()</slot>
  <slot>slot1()</slot>
  <slot>onTypologyClick()</slot>
  <slot>onAmericanClick()</slot>
  <slot>onShowClick()</slot>
  <slot>onNationalClick()</slot>
  <slot>onStadiumsClick()</slot>
  <slot>onTeamClick()</slot>
  <slot>onSearchClick()</slot>
  <slot>onRoofClick()</slot>
  <slot>onDateClick()</slot>
  <slot>onSeatingClick()</slot>
  <slot>onGreatClick()</slot>
  <slot>onLeastClick()</slot>
  <slot>onGraphInfoClick()</slot>
  <slot>onStartTrip()</slot>
 </slots>
</ui>
